src/s_conf.c

```diff
@@ -6839,6 +6840,33 @@ int	_conf_set(ConfigFile *conf, ConfigEntry *ce)
 		else if (!strcmp(cep->ce_varname, "level-on-join")) {
 			tempiConf.level_on_join = channellevel_to_int(cep->ce_vardata);
 		}
+		/* setting "all" will enforce static-quit and static-part to all users
+		 * 
+		 * setting "unregistered" will enforce static-quit and static-part to
+		 * unregistered users only -dboyz
+		 * TODO: Produce error message by adding else
+		 */
+		else if (!strcmp(cep->ce_varname, "static-quit-part-set")) {
```

Comment:
how about static-quit-part-users instead, since we're not "setting" it, we're setting which users it impacts?